people-on-near.json

```diff
@@ -1,4 +1,32 @@
 [
+    {
+        "name": "Mister",
+        "email": "",
+        "preferredContact": "X",
+        "organization": "HOT Labs",
+        "team": "",
+        "jobTitle": "Co-founder",
+        "description": "Co-founder & CEO 'HOT Labs'",
```

Comment:
Are both Mister and Peter the CEOs of Hot Labs?

Reply:
i think Mister is the co founder

Reply:
I see. We may need to update the description I think.